dracut/30ignition/coreos-teardown-initramfs-network.service

```diff
@@ -26,6 +26,13 @@ Before=initrd-switch-root.target
 OnFailure=emergency.target
 OnFailureJobMode=isolate
 
+# If we are already heading towards emergency.target
+# then don't try to stop this unit because it will fail
+# when trying to access files in /sysroot/etc/. The failure
+# is mostly harmelss but having the extra error messages
```

Comment:
harmless

Reply:
Could do this as a followup though - just a harmless comment about a harmless comment about a harmless comment.

Reply:
fixed the harmless comment 😜